frontend/src/components/print/print-nuxt/DownloadNuxtPdfListItem.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <v-list-item @click="generatePdf">
+    <v-list-item-icon>
+      <v-icon v-if="loading" class="mdi-spin">mdi-loading</v-icon>
+      <v-icon v-else>mdi-nuxt</v-icon>
+    </v-list-item-icon>
+    <v-list-item-title>
+      {{ $tc("components.print.pdfDownload.nuxt.label") }}
```

Comment:
https://github.com/ecamp/ecamp3/wiki/localization-and-translations#translation-file-structure

The translation keys must match the file structure. This one should be `components.print.printNuxt.downloadNuxtPdfListItem.label`.
This was our agreement when we decided not to use component-local translations. The idea is that this way we can easily find out where a translation is used, delete unused translations and maybe some day even introduce automatic prefixing such that you could just use the translation key `.label` here.

Reply:
Okay, I'll change it in the other components as well.

Reply:
Nevermind, I'll remove this button anyway soon. All other print components do not abide to those rules.

Reply:
Fixed in 27ee19e